### DISCUSSION TITLE
Download issues

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#88] [Feedback / Issues]
Hi team,

We are currently trying to download the Hire Agreement and Mobe Freight Brief and they are both downloading as Job Summaries.

Can you please review.

**Comment by twrayden** [comment]
Hi Wade, thanks for reporting, will take a look at this now and update here when fixed.

**Comment by twrayden** [comment]
This has now been fixed, you will likely need to refresh your page to get the latest update.
